src/main/java/org/example/lesson33/lesson30interface/StackIsEmptyException.java

```diff
@@ -0,0 +1,7 @@
+package org.example.lesson33.lesson30interface;
+
+public class StackIsEmptyException extends Exception{
+    public StackIsEmptyException() {
+        super ("Stack Is Empty");
+    }
+}
```

Comment:
Виключення які виникають при роботі зі стеком мають бути `unchecked`. Вам потрібно наслідувати не від `Excpetion` а від `RuntimeExcpetion`.
```suggestion
public class StackIsEmptyException extends RuntimeException {
    public StackIsEmptyException() {
        super ("Stack Is Empty");
    }
}
```